pkg/apis/wavefront/v1alpha1/wavefrontcollector_types.go

```diff
@@ -37,6 +37,15 @@ type WavefrontCollectorSpec struct {
 	// For pinning Collector to a specific version, you will need to set this option to false.
 	// We support only minor version Auto Upgrades.
 	EnableAutoUpgrade bool `json:"enableAutoUpgrade,omitempty"`
+
+	// Openshift Specific configurations starts from here.
+
+	// Set to true when running collector in Openshift platform.
+	Openshift bool `json:"openshift,omitempty"`
+
+	// If set to true, Collector will use default config bundled in the image
+	// else it will use the config from ConfigName
+	OpenshiftDefault bool `json:"openshiftDefault,omitempty"`
```

Comment:
nit: UseOpenshiftDefaultConfig is probably more descriptive ?
for collector to work when no configName is specified , is setting this to true by the user mandatory ? or can we just assume this is true internally in the code when there is no configName ?

Reply:
When running operator in k8's case without configName may error out with misleading error message. I guess Keeping this flag would be useful.

Reply:
Changed config property name.